.gitlab-ci.yml

```diff
@@ -11,4 +11,4 @@ variables:
 tox:
   parallel:
     matrix:
-      - LANGUAGE_VERSION: [python3.8, python3.9]
```

Comment:
It looks like the inclusion of python3.9 was not a recent change. The py39 tests did, however, start failing with the previous commit: https://github.com/NCI-GDC/gdc-client/commit/2af71a5d042a78f9f60b15f2638446a1292e5443

Do we know what broke it? If not and if not urgent, should we perhaps take the time to investigate and try to correct?

Reply:
yep, i have more tickets in the detail. I'm assuming we switched gitlab runners to a FIPs runner, and we are getting bit by the md5 issue and the fix for py39 is to set the usedforsecurity=False. I don't think its supported in py38 though, and instead of having to support both versions of python in code: 1) we need to actually run pyupgrade to py39 for gdc-client, which I don't believe we've done and 2) we will need to upgrade to py39 soon and drop py38 support, so the fix will be easier when we make that change.

Reply:
Sounds good and makes sense to me, thank you for the feedback